src/domains/ingredients/ingredients.service.ts

```diff
@@ -21,4 +24,9 @@ export class IngredientsService implements IIngredientsService {
     );
     return Promise.all(ingredients);
   }
+
+  async all() {
```

Comment:
```suggestion
  async getAll() {
```

Kanske bättre att vara mer explicit med vad som händer?